docs/pages/get-started/installation.md

```diff
@@ -2,52 +2,51 @@
-- **Git**: Additionally, you'll need Git to create new projects. [You can download Git from here](https://git-scm.com).
+- [Node.js 12](https://nodejs.org/en/) or greater
+- [Git](https://git-scm.com) for version control
+- [Watchman](https://facebook.github.io/watchman/docs/install#buildinstall) for MacOS users
 
 ### Installing Expo CLI
 
-We recommend installing Expo CLI globally, you can do this by running the following command:
-
-```
-npm install -g expo-cli
-```
-
-Verify that the installation was successful by running `expo whoami`. You're not logged in yet, so you will see "Not logged in". You can create an account by running `expo register` if you like, or if you have one already run `expo login`, but you also don't need an account to get started.
+The best way to get started with universal React is by using `expo-cli`, alternatively you can use [`create-react-native-app`](https://github.com/expo/create-react-native-app) for more templates.
 
-> 😳 **Running into problems?** Try searching for your error message on the [forums](https://forums.expo.io) &mdash; you're probably not the first person to encounter your issue, and the forums are a great resource for these types of problems.
+<TerminalBlock cmd={['# Install the command line tools', 'npm install --global expo-cli','', '# Create a new project', 'expo init my-project', 'cd my-project']} />
 
-### Optional: Installing Watchman
+> 😳 **Need help?** Try searching the [forums](https://forums.expo.io) &mdash; which are a great resource for troubleshooting.
 
-Some macOS users encounter issues if they do not have Watchman installed on their machine, so if you are using a Mac we recommend that you install it. [Download and install Watchman](https://facebook.github.io/watchman/docs/install#buildinstall).
+<TerminalBlock cmd={['# Start the dev server', 'expo start']} />
 
-> 💡Watchman watches files and records when they change, then triggers actions in response to this, and it's used internally by React Native. 
+- Pressing `i` will open in the [iOS Simulator](../../workflow/ios-simulator/).
+- Pressing `a` will open in the [Android Emulator](../../workflow/android-studio-emulator/).
+- Pressing `w` will open in your Browser. Expo supports all major browsers.
 
 ## 2. Mobile app: Expo client for iOS and Android
 
-Expo client is the tool you will use to run your projects while you're developing them. When you serve your project with Expo CLI, it generates a development URL that you can open in Expo client to preview your app.
+The fastest way to run your project is with the Expo client app on a real iOS or Android device. Develop and test your projects on a physical device with the [Expo client app](https://expo.io/tools). When a project is `start`ed with Expo CLI, it can be opened in the Expo client.
```

Comment:
```suggestion
When a project is `start`ed with Expo CLI, it can be opened in the [Expo client app](https://expo.io/tools). This is, without a doubt, the fastest way to run and test your project on a real iOS or Android device.
```